Source: http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/media-libs/libmng/files/libmng-2.0.2-stdio.patch
Upstream: no
Reason: compile failure:
    /usr/include/jpeglib.h:954:30: error: unknown type name 'FILE'
     EXTERN(void) jpeg_stdio_dest JPP((j_compress_ptr cinfo, FILE * outfile));

diff -ruN libmng-2.0.2.org/libmng_types.h libmng-2.0.2/libmng_types.h
--- libmng-2.0.2.org/libmng_types.h	2013-01-20 04:45:24.000000000 +0100
+++ libmng-2.0.2/libmng_types.h	2014-11-18 20:45:59.815767867 +0100
@@ -204,6 +204,7 @@
 #define HAVE_BOOLEAN
 typedef int boolean;
 #endif
+#include <stdio.h>
 #include <jpeglib.h>
 #endif /* MNG_INCLUDE_IJG6B */
 
